Add pull request and issue templates (#3)
openrewrite/.github: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: ''
labels: 'bug'
assignees: ''
---
<!--
Thank you for reporting an issue with OpenRewrite!
We appreciate you taking the time to help us improve.
Please fill out the template below to help us understand and reproduce the issue.
Feel free to delete any sections that don't apply to your issue.
-->

## What version of OpenRewrite are you using?
<!--
Whenever possible please try to replicate your issue with the latest versions of OpenRewrite.
The latest versions of some common OpenRewrite projects are listed here.

> - https://github.com/openrewrite/rewrite-maven-plugin/releases

Reviewer: We could consider adding a link to our docs which has the latest versions _usually_: https://docs.openrewrite.org/reference/latest-versions-of-every-openrewrite-module

- https://github.com/openrewrite/rewrite-gradle-plugin/releases
- https://github.com/openrewrite/rewrite/releases
- https://github.com/openrewrite/rewrite-recipe-bom/releases
- https://github.com/openrewrite/rewrite-spring/releases
- https://github.com/openrewrite/rewrite-static-analysis/releases

We release every few weeks, so it's possible that your issue has already been fixed.

> You might even want to use [our snapshot releases](https://docs.openrewrite.org/reference/snapshot-instructions) with recent changes.

Reviewer: ```suggestion

If you want to try the most recent changes that haven't been fully released yet, you can check out our [snapshot releases](https://docs.openrewrite.org/reference/snapshot-instructions)
```

-->
I am using

- OpenRewrite v1.2.3
- Maven/Gradle plugin v1.2.3
- rewrite-_module_ v1.2.3

## How are you running OpenRewrite?
<!--

> Are you using the Maven plugin, Gradle plugin, Moderne CLI, Moderne.io or some other way?

Reviewer: ```suggestion
Are you using the Maven plugin, Gradle plugin, Moderne CLI, Moderne.io or something else?
```

Is your project a single module or a multi-module project?

> Can you share your configuration such that we can rule out any configuration issues?

Reviewer: ```suggestion
Can you share your configuration so that we can rule out any configuration issues?
```
